spec/cucumber/formatter/junit_spec.rb

```diff
@@ -69,9 +69,9 @@ def after_step(step)
           FEATURE
 
           it 'raises an exception' do
-            expect(-> {
+            expect(-> do
               run_defined_feature
-            }).to raise_error(Junit::UnNamedFeatureError)
+            end).to raise_error(Junit::UnNamedFeatureError)
```

Comment:
Oh, this would _also_ read easier to me as a single-line

```ruby
expect(->{ run_defined_feature }).to raise_error(Junit::UnNamedFeatureError)
```